### ISSUE TITLE
Obtain reference profile

### ISSUE THREAD

**Issue by lucreziaPatruno** [YosefLab/SymSim#20]
Hello,
I would like to ask you if there is a way to obtain a reference profile for each cell population simulated in SymSim. The main problem is the following: I use SimulateTrueCounts to simulate three discrete sub-populations, with a Sigma=0.5. My initial idea was that of setting Sigma=0 in a new simulation (using the same seed) to get a reference profile (i.e. one unique expression profile) for each population, but I observed that the profiles obtained in the last simulation are not correlated with those obtained using Sigma=0.5.

Is there a way to get these reference profiles from a simulation, such as using its kinetic parameters?
Thank you
Lucrezia

**Comment by xiuweiz** [comment]
Hi Lucrezia,

The way we simulate these populations is that for each cell we first simulate its EVF vector. The Simga value controls the deviation of the EVFs from the mean EVFs of the population. After we obtain the EVF vector of a cell, we calculate the corresponding kinetic parameters of each gene in this cell, then sample the gene-expression level of the gene in the cell from the kinetic model. So extra stochasticity is introduced during this sampling procedure. Using "expected" gene-expression levels calculated directly from kinetic parameters (as you also mentioned) can be one way to reduce this stochasticity. We have used the kinetic parameters to calculate expected gene-expression levels when doing the DE benchmarking. In the paper we mentioned how we can calculate it:
![image](https://user-images.githubusercontent.com/22404227/88719073-b4119100-d0f0-11ea-932a-c54b960e532b.png)
If you set sigma=0 you should get the same EVF vectors for all cells in one cluster then hence the same kinetic parameters? Then you can use what is described in the screenshot to calculate expected gene-expression profiles. 

Please let me know if there are further questions.

Xiuwei

**Comment by lucreziaPatruno** [comment]
Hello Xiuwei,
Thank you for your answer. We have done what you suggested to obtain the expected gene expression profiles. However, if we compare each cell obtained using Sigma=0.5 and the expected profile of its population obtained with Sigma=0 we don't obtain highly correlated values. For example, if we compute correlation between each single cell in pop1 and the expected profile of pop1, and then we compute correlation between cells in pop1 and the expected profile of pop2 we don't get higher values (i.e. Spearman Rho) in the first case as we would expect.
Thank you, 
Lucrezia

**Comment by xiuweiz** [comment]
Did you run SimulateTrueCounts() separately with Sigma=0.5 and Sigma=0? If they are two separate runs then it's possible that there are little correlation between the two runs as each run is performed stochastically.

**Comment by lucreziaPatruno** [comment]
Hello xiuweiz, 
we run a simulation with Sigma=0.5, then we decreased the value of sigma (e.g.: 0.1, 0.01, 0.001) and we finally used sigma=0. Each time we used the same seed for the simulations, so that if we run two distinct experiments with the same sigma we get the exact same results. We have observed that while there is a correlation between profiles obtained with sigma=0.5 and Sigma=0.1,...,0.001, this correlation is not observed anymore when using sigma=0.00001 and lower. However, we solved our problem by using the lowest possible Sigma for which we observe some correlation, and taking the mean value for the kinetic parameters for each sub-population. In this way we can use the obtained mean values to calculate the expected gene expression profile.

Thank you, 
Lucrezia
